docs/getting-started/experiments/mabs/index.md

```diff
@@ -0,0 +1,93 @@
+# Multi-Armed Bandits
+
+Multi-armed Bandits (MABs) are useful for running experiments where you have multiple variants of a feature / implementation that you want to test, and want to converge to answer quickly about which variant produces the best results.
+
+## What are Multi-Armed Bandits (MABs)?
+
+MABs are a specialized reinforcement learning algorithm: let's imagine that you have set up $N$ variants of an experiment, and for each variant you assign some corresponding probability for getting a desired result. You serve your each of your beneficiaries one of these variants (the strategy for choosing the variant is based on the corresponding probabilities), and observe theresult of their interaction with it: for example, if you are experimenting with a website interface, you might want to track how long a user spends on the website.
+
+Once you have observed the result, you can now update your arm / variant's probability of achieving the desired result. The next time you serve a
+beneficiary one of the variants, you can use the updated probabilities to determine which variant to show them.
+
+Your precise strategy for using the probabilities to choose variants to serve can be optimized for observing results for as many variants as possible (exploration) or converging to the best variant quickly (exploitation). *Our current implementation supports one algorithm (Thompson sampling) that balances both exploration and exploitation, but we're working on implementing more!*
```

Comment:
```suggestion
The precise strategy for using the probabilities to choose variants to serve can be optimized for observing results for as many variants as possible (exploration) or converging to the best variant quickly (exploitation). *Our current implementation supports one algorithm (Thompson sampling) that balances both exploration and exploitation, but we're working on implementing more!*
```

Reply:
I suggest we scrap this or make this a lot simpler. It's a little confusing.

I'm thinking that the audience is either "economist types" - they understand stats but not these methods; or "product owners" who will be totally lost.